Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -22,6 +22,7 @@ public function configDataProvider(): array
                 [
                     'namespace' => 'myapp',
                     'type' => 'in_memory',
+                    'storage' => ['type' => 'in_memory'],
```

Comment:
I assume this is a test for the legacy use case. If so, why do we need it here?

Reply:
I let the configuration manage the legacy part. All values are inject into `storage` and the extension use `storage` it.
This behavior is internal and it will be easier to remove those deprecated parameters. I can remove this and let the extension manage the legacy but as I said it is internal.